js/clinicalTimeline.js

```diff
@@ -163,16 +152,24 @@ var clinicalTimeline = (function(){
       .attr("width", overviewAxisWidth)
       .attr("class", "overview");
 
-    clinicalTimelineOverviewAxis(overviewSVG, getTickValues, minDays, maxDays, overviewAxisWidth, formatTime, daysToTimeObject, getZoomLevel, width,  margin);
-    clinicalTimelineVerticalLine(timelineBool.enableVerticalLine, beginning, ending, timelineBool.advancedView, timelineBool.tooltipOnVerticalLine);
-    initHelperLines(timelineBool.enableHelperLines, zoomFactor, timelineBool.advancedView, divId, formatTime, daysToTimeObject);
-
-    if(timelineBool.advancedView){
-      initAdvancedView(overviewSVG)
-    } else {
-      initSimpleView(overviewSVG);
+    //object to be shared by all plugins
+    clinicalTimelineVar = {
```

Comment:
Can we get this object as a function on `timeline`? That way you don't have to separately pass it to the plugins. Also `clinicalTimelineVar`as a name seems a little generic. Are these `read-only` variables? Maybe you can call them `clinicalTimelineVarReadOnly`? Or is there a better name?